### DISCUSSION TITLE
integrazione Amazon Firestick

### DISCUSSION THREAD

**Discussion by fenir52** [indomus/forum#2091] [GENERALE - Discussioni generiche]
Salve a tutti

premetto sono nuovo e sto cercando di capirci un pò di cose, ho installato su un raspberry pi 3+ HA core su Docker, cercando in rete tutte le guide per  per integrare la Firestick parlano di installare un server ADB tramite Supervised, ovviamente non presente perché ho installato la versione HA Core in questo caso che soluzione devo intraprendere?
Ci sta la possibilità di integrare Supervised o si può fare in un altro modo?

se ci sta qualche anima pia che mi riesce a indirizzare sulla strada giusta sarebbe stupendo.

**Comment by NdR91** [comment]
Cosa intendi per "integrare Firestick"?

I dispositivi Alexa, in genere, si controllano tramite [Alexa Media Player](https://github.com/custom-components/alexa_media_player)

**Comment by fenir52** [comment]
accendere o spegnere la tv attaccata alla firestick, avviare netflix, ecc... 
Provo con Alexa Media Player che non conoscevo 

Grazie
